apps/backend/src/transaction/dto/transaction.dto.ts

```diff
@@ -16,6 +16,10 @@ import { AccountingDTO } from './accounting.dto';
 import { PaymentDTO } from './payment.dto';
 import { SourceDTO } from './source.dto';
 
+/**
+ * Potentially deprecated, but will likely still need some formatting
```

Comment:
Is there a Jira issue we can associate this comment with? Thinking we'll want a quick ref to trigger a review here when we pick up whatever work would be using this formatting

Reply:
I think this'll come as part of the API implementation, when we start doing up endpoints for transactions. We can work to combine the two. Should I link it to https://bcdevex.atlassian.net/browse/CCFPCM-460 for now?